### ISSUE TITLE
[a11y]: Select -> value is announced twice by screen reader

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon#20457]
### Package

@carbon/react

### Browser

Chrome

### Operating System

Windows

### Package version

Latest

### React version

Latest

### Automated testing tool and ruleset

Manually

### Assistive technology

JAWS / NVDA

### Description

If you use a screen reader, and go to the Select component:

https://react.carbondesignsystem.com/?path=/story/components-select--default&args=helperText:

If you have no helper text, the screen reader reads the value twice:

`Select an option  combo box  Option 2  collapsed  Option 2`

Here's full AVT report:

```
Component: Billing method combobox

Issue description: Screen reader announces the billing method combobox name twice and position of the combobox items are not announced. Note :Issue exists in bare metal server as well

Recommended fix: Screen reader should announce the information only once so that user will not get confused.

User impact: If redundant or duplicate information is announced twice by a screen reader, it can cause confusion and frustration for users who rely on the screen reader to navigate and interact with the content.

WCAG Mapping: 1.3.1 Info and Relationships

Level of Compliance: A

Severity: Medium

Test tool: JAWS_Chrome
```

### WCAG 2.1 Violation

WCAG Mapping: 1.3.1 Info and Relationships

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-select--default&args=helperText:

### Steps to reproduce

go to the story book link above, and hit tab on keyboard until you land on the select dropdown.

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Thanks for opening this issue. I’ve confirmed and we tested it together with the accessibility team, and it really looks like the issue has been resolved, we weren’t able to reproduce it. For this reason, we are closing the issue.
